pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -804,6 +805,42 @@ func (nc *Controller) monitorNodeHealth() error {
 	return nil
 }
 
+// labelNodeDisruptionExclusion is a label on nodes that controls whether they are
+// excluded from being considered for disruption checks by the node controller.
+const labelNodeDisruptionExclusion = "node.kubernetes.io/exclude-disruption"
```

Comment:
Shouldn't this be in a publicly exported place?

Reply:
Until it hits beta, I would say no.  Referencing constants across multiple concrete domains (controller manager / rest of kube) would be a bug.